site.rb

```diff
@@ -0,0 +1,93 @@
+RACK_ENV ||= ENV["RACK_ENV"] ||= "development" unless defined?(RACK_ENV)
+
+require "rubygems" unless defined?(Gem)
+require "bundler/setup"
+Bundler.require(:default, RACK_ENV)
+
+require "json"
+require "open-uri"
+require "rss"
+require "set"
+Dir[File.dirname(__FILE__) + '/lib/*.rb'].each {|file| require file }
```

Comment:
Space between { and | missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.